python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -91,6 +93,60 @@ def _build_fil_classifier(m, data, arg={}, tmpdir=None):
                   storage_type=arg["storage_type"])
 
 
+def _build_fil_skl_classifier(m, data, arg={}, tmpdir=None):
+    """Setup function for FIL SKLearn classification benchmarking"""
+
+    train_data, train_label = _training_data_to_numpy(data[0], data[1])
+
+    
+
+    params = {
+        "n_estimators": 10,
+        #    "max_depth": 16,
+        "max_leaf_nodes": 2**10,
+        "max_features": "sqrt",
+        "n_jobs": -1,
+        "random_state": 42,
+    }
+    params.update(arg)
+    
+    # remove keyword arguments not understood by SKLearn
+    params.pop("fil_algo", None)
```

Comment:
not sure you need all these `None` second parameters since you're not using the return value. Personally, i'd do a compressed python style like:
 for param_name in ["output_class", "threshold", "storage_type", ...]:
  params.pop(param_name)

Reply:
Done.

`None` in `params.pop()` is necessary, as it otherwise throws an exception if a key is not present: https://docs.python.org/3/library/stdtypes.html#dict.pop.